b2sdk/transfer/inbound/downloader/abstract.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         max_chunk_size: int | None = None,
         align_factor: int | None = None,
         check_hash: bool = True,
+        retry_time: int = 5,
```

Comment:
the default of 5 here is inconsistent with the default of 300 higher up, also 5s is practically nothing for a download operation